include/mongoose.hrl

```diff
@@ -34,3 +34,11 @@
 %% ---------------------------------
 %% Logging mechanism
 -include("mongoose_logger.hrl").
+
+%% for worker_pool
+-record(pool, {name :: atom(),
+    selection_strategy :: atom(),
+    path_prefix :: binary(),
```

Comment:
The prefix is very `http` specific and we use this record for not only `http` pools. I suggest to create a field `extra` where such pool-specific like `path` could be stored.